src/messageReactionAdd.ts

```diff
@@ -60,6 +61,10 @@ export const messageReactionAdd = async (reaction: MessageReaction) => {
 
         // if message owner gets 5+ reactions add "high value" role
         if (reactionsCount >= REACTIONS_COUNT) {
+            const isAssignedRole = (await getUserRoles(reaction.message.member!)).includes(ROLE.HIGH_VALUE.name);
+
+            if(isAssignedRole) return;
```

Comment:
We always use brackets, I think it would be good to be consistent here, even if a bit more verbose.

```suggestion
            if(isAssignedRole) {
               return;
            }
```

Reply:
Please add space after `if` keyword 😁

Reply:
No worries. I  was contemplating brackets vs none.

Reply:
Thanks @alrifay and @AllanRegush 👍